File: sololearn/NoNumerals/NoNumerals.py
# input string as splited array
in_str = str(input()).split()
# init out string
out_str = ""
# num array to convert index to number [0-10]
num_arr = ["zero", "one", "two", "three", "four", "five",
"six", "seven", "eight", "nine", "ten"]

# loop to check for int & convert if in b/w [0-10]
for word in in_str:
    if word.isdigit():
        num = int(word)
        # convert to numeral if <= 10
        if num <= 10:
            word = num_arr[num]
    # generate out string
    out_str += " " + word

# print the new string removing starting space
print(out_str.strip())

File: sololearn/NoNumerals/NoNumerals.cpp
/*
Author: chankruze (chankruze@geekofia.in)
Created: Mon Aug 10 2020 18:01:52 GMT+0530 (India Standard Time)

Copyright (c) Geekofia 2020 and beyond
*/

#include <bits/stdc++.h>

#include <algorithm>
#include <iostream>
#include <string>
#include <vector>

using namespace std;

bool checkNumber(string str) {
    for (int i = 0; i < str.length(); i++)
        if (isdigit(str[i]) == false)
            return false;
    return true;
}

int main(int argc, char const *argv[]) {
    string txt, word;
    vector<string> numerals = {"zero", "one", "two", "three", "four", "five",
                               "six", "seven", "eight", "nine", "ten"};

    getline(cin >> ws, txt);

    stringstream ss(txt);

    while (getline(ss >> ws, word, ' ')) {
        if (checkNumber(word)) {
            int num = stoi(word);
            if (num <= 10) {
                cout << numerals[stoi(word)];
            } else {
                cout << num;
            }
        } else {
            cout << word;
        }
        cout << " ";
    }
}
